proc/proc_windows.go

```diff
@@ -112,7 +117,7 @@ func newDebugProcess(pid int, exepath string) (*Process, error) {
 	var err error
 	var tid, exitCode int
 	dbp.execPtraceFunc(func() {
-		tid, exitCode, err = dbp.waitForDebugEvent()
+		tid, exitCode, err = dbp.waitForDebugEvent(true, false)
```

Comment:
nit: Passing booleans in as parameters makes it hard to know what they mean when reading at the caller. Would do you think about named constants or using a bitmap flag? E.g. `opts := WAIT_NONBLOCK | SUSPEND_NEW_THREADS`?